CHANGELOG.md

```diff
@@ -69,3 +69,5 @@
 ## [v1.1.15](https://github.com/BoltApp/bolt-magento2/releases/tag/1.1.15) 2019-06-12
  - Fixes for multi-store backend support.
  - Option to toggle emulated session in api calls
+## [v2.0.0](https://github.com/BoltApp/bolt-magento2/releases/tag/2.0.0) 2019-07-02
```

Comment:
is this really the PR that introduces this? if this is the case, please can we break it up so that it is easier to review. 

overall as the number of files/lines grows higher the chances of a reviewer missing something is high. Especially given that this is a bog feature and updating plugin versions is hard i think we should be especially careful